File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Models/Dtos/FacilityConfigEditModel.cs
using Cen.Wms.Client.Models.Enums;

namespace Cen.Wms.Client.Models.Dtos
{
    public class FacilityConfigEditModel
    {
        public AcceptanceProcessType AcceptanceProcessType { get; set; }
        public string PalletCodePrefix { get; set; }
        public bool IsAcceptanceByPapersEnabled { get; set; }
        // public int TZBias { get; set; }
        // public int MaxExpirationDaysPlus { get; set; }
    }
}

File: TCDNew-master/src/ui.android/lib/models/dtos/FacilityConfigEditModel.dart
import 'package:tiq/models/enums/AcceptanceProcessType.dart';

class FacilityConfigEditModel {
  AcceptanceProcessType acceptanceProcessType;
  String palletCodePrefix;
  bool isAcceptanceByPapersEnabled;

  FacilityConfigEditModel(
    this.acceptanceProcessType,
    this.palletCodePrefix,
    this.isAcceptanceByPapersEnabled
  );

  factory FacilityConfigEditModel.fromJson(Map<String, dynamic> json) {
    FacilityConfigEditModel data;

    if (json != null) {
      data = FacilityConfigEditModel(
          json['acceptanceProcessType'] == 1000
              ? AcceptanceProcessType.simple
              : json['acceptanceProcessType'] == 2000
                ? AcceptanceProcessType.palletized
                : throw FormatException('Incorrect JSON'),
          json['palletCodePrefix'],
          json['isAcceptanceByPapersEnabled']
      );
    }

    return data;
  }
}